lib/context.js

```diff
@@ -58,6 +61,28 @@ class Context {
   get isBot() {
     return this.payload.sender.type === 'Bot';
   }
+
+  /**
+   * Reads the plugin configuration from the given YAML file in the `.github`
+   * directory of the repository.
+   *
+   * @example
+   *
+    // Load config from .github/myplugin.yml in the repository
+    const config = await context.config('myplugin.yml');
+
+    if(config.close) {
+      context.github.issues.edit(context.issue({state: 'closed'}));
+    }
+   *
+   * @param {string} fileName Name of the YAML file in the `.github` directory
+   * @return {Promise<Object>} Configuration object read from the file
+   */
+  async config(fileName) {
+    const params = this.repo({path: path.join('.github', fileName)});
```

Comment:
I think adding an optional `basePath` parameter is a great idea! ✨ My main issue is that if you want to go outside the convention, you couldn't use this built in method at all and would have to write your own. So adding that definitely addresses my concerns